macros/storage/boot-on-block.mdx

```diff
@@ -0,0 +1,5 @@
+---
+macro: storage-boot-on-block
+---
+
+The boot-on-block feature allows Instances to boot from attached [Block Storage](#block-volumes) volumes instead of local volumes. This feature is available for all Instance types. When creating an Instance, you can select Block Storage as a system volume type. This allows you to reboot Instances from the block volume on which your OS is stored. 
```

Comment:
```suggestion
The boot-on-block feature allows Instances to boot from attached [Block Storage](#block-volumes) volumes instead of local volumes. This feature is available for all Instance types.
```